docs/releases.rst

```diff
@@ -1,6 +1,170 @@
+
+Download zonefile on domain details page
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+On the domain details page a download option is added for downloading the zone file. This zone file can be used to easily import all Mailu DNS settings.
+
+Roundcube spellchecker
+^^^^^^^^^^^^^^^^^^^^^^
+The roundcube spellchecker can be configured to support different languages than English. For how to configure alternative languages, refer to the FAQ entry: `How can I add more languages to roundcube's spellchecker`.
+
+Improved mailbox indexer and full attachment indexing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The dovecot indexer indexes all emails for the server-side search functionality. It is used when searching via the webmail client. In email clients it might be required to enable server-side searching.
+
+The dovecot indexer has been switched from fts-xapian to fts-flatcurve. In the future this will be the new default indexer for dovecot. This indexer is quicker and results in smaller index files.
+
+Apache Tika has been added to Mailu to add support for attachments indexing. As a result the server-side search functionality does not only search through emails, but also through the attachments of emails.
+
+Refer to `FULL_TEXT_SEARCH` in the `configuration reference` to enable indexing for non-English languages.
+
+After upgrading, the new indexes are not automatically created. To create these after upgrading Mailu:
+
+From `bash` run:
+
+.. code-block:: bash
+
+    find /mailu/mail -type d -name xapian-indexes -prune -exec rm -r {} \+
+
+Via docker compose run (to force reindexing):
+
+.. code-block:: bash
+
+    docker compose exec imap doveadm fts rescan -A
+    docker compose exec imap doveadm user '*'|while read u; do docker compose exec imap doveadm index -u $u '*'; done
+
+
+Introduction AUTH_REQUIRE_TOKENS
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The environment variable `AUTH_REQUIRE_TOKENS` has been introduced. This setting can be enabled to force  email clients to use authentication tokens (instead of passwords) for authenticating to Mailu. Note that authentication tokens can now also be generated via the RESTful API.
+
+It is recommended to use authentication tokens instead of passwords for connecting email clients to Mailu.
+
+
+Change in behaviour
+```````````````````
+
+POSTFIX_LOG_FILE removed
+^^^^^^^^^^^^^^^^^^^^^^^^
+The setting POSTFIX_LOG_FILE and its functionality has been removed from Mailu. To log to file (for any container) it is possible to use journald and rsyslogd.
+
+The new FAQ entry `How can I view and export the logs of a Mailu container?` describes how log files can be viewed via journald. It also provides instructions for how to install and configure rsyslogd for saving container logs to file system (including log rotation).
+
+The hardened malloc is disabled by default
```

Comment:
That heading is wrong. It is not disabled by default... we just enable it later in the process once we have ensured the CPU can handle it.

For optimal protection we do recommend that users set the variable so that it gets enabled sooner that it would otherwise be.